airflow/example_dags/example_task_group_decorator.py

```diff
@@ -57,8 +57,8 @@ def task_end():
 
 
 # Creating TaskGroups
-@task_group
-def task_group_function(value):
+@task_group()
```

Comment:
Not sure if our task_group decorator also suppots () syntax ?

Reply:
It does

Reply:
Cool

Reply:
It does, but generally in the examples I think we should use it without parentheses

Reply:
MyPy complans here if I do not use parentheses. I will double check though

Reply:
```
airflow/example_dags/example_task_group_decorator.py:73: error: Value of type
variable "T" of "task_group_function" cannot be "int"
            current_task_group = task_group_function(i)
```

Reply:
I think `task_group`’s `if callable(python_callable)` case needs a `cast`.

Reply:
Oh it’s more complicated than that. I’ll just push a solution to your branch 🙂

Reply:
Done